README.md

```diff
@@ -1,8 +1,106 @@
-# Cabrillo_ROV_2023
+# Cabrillo ROV 2023
+
 Cabrillo College Robotics Club repo for the MATE ROV 2023 competition
 
-# Software
+## Software
 
 ROS 2 Humble Hawksbill (Ubuntu 22.04 LTS)
 
 [ROS Documentation](http://docs.ros.org/en/humble/index.html)
+
+## Building 
+
+The ROS nodes are built with the `colcon` program. A `makefile` is present to help me remember what the `colcon` commands are. Build the project with:
+
+```console 
+$ make 
+``` 
+
+The built artifacts are in:
+
+- `build` 
+- `install`
+- `log` 
+
+Clean the build with:
+
+```console
+$ make clean 
+``` 
+
+## Launching all Nodes 
+
+There are launch configurations in the `launch` directory. To run all the configured nodes on the ROV:
+
+```console 
+ros2 launch ./launch/rov.yaml  
+```
+
+To launch all the nodes on the laptop:
+
+```console 
+ros2 launch ./launch/laptop.yaml  
+```
+
+## Running Individual Nodes 
+
+The built nodes are a part of a local build area. Source the following script to setup your paths:
+
+```console 
+$ source install/setup.bash # Choose one that matches your shell.
+``` 
```

Comment:
we should probably add ansible to add this to the .bash_profile on devkit category devices